spring/README.md

```diff
@@ -32,13 +50,34 @@ import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
-@ComponentScan("com.example.app")
+@ComponentScan("cucumber.runtime.java.spring.beans")
 public class Config {
     // the rest of your configuration
 }
 ```
 
 * Add `@DirtiesContext` to your test configuration if each scenario should have a fresh application context.
+```java
+package cucumber.runtime.java.spring.dirtiescontextconfig;
+
+import com.example.app.Config;
+import cucumber.runtime.java.spring.beans.Belly;
+import cucumber.runtime.java.spring.beans.BellyBean;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ContextConfiguration;
+
+@ContextConfiguration(classes = { Config.class })
+@DirtiesContext
+public class DirtiesContextBellyStepDefs {
```

Comment:
To keep the documentation consistent this whole class should be similar to `SomeServiceSteps` with ` @DirtiesContext` added.